pkg/controllers/git/git.go

```diff
@@ -384,7 +384,7 @@ func (h *handler) OnChange(gitrepo *fleet.GitRepo, status fleet.GitRepoStatus) (
 	}
 	status.Resources, status.ResourceErrors = h.display.Render(gitrepo.Namespace, gitrepo.Name, bundleErrorState)
 	status = countResources(status)
-	volumes, volumeMounts := volumes(gitrepo, configMap)
+	volumes, volumeMounts := volumes(h, gitrepo, configMap)
```

Comment:
I wonder if this should be `h.volume(` now. Or we could pass in `h.secrets`, since that's the only thing we seem to use?

Reply:
Good catch, refactored this to pass in `h.secrets`. Thanks!